### PR TITLE
Internal: Home Screen - Create 'Add-ons' section [ED-13986]

### PR DESCRIPTION
**Pull request by elchugreeva**

### CODE REVIEW

File: modules/home/assets/js/components/addons-section.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import Button from '@elementor/ui/Button';
+import Card from '@elementor/ui/Card';
+import CardActions from '@elementor/ui/CardActions';
+import CardContent from '@elementor/ui/CardContent';
+import CardMedia from '@elementor/ui/CardMedia';
+
+const Addons = ( { ...props } ) => {
+	const addonsArray = props.addonsData.repeater.filter( ( item ) => false === item.is_installed );
+	const cardsPerRow = 3 === addonsArray.length ? 3 : 2;
+	return (
```
Comment (hein-obox) [inline modules/home/assets/js/components/addons-section.js:13]: Empty line.

File: modules/home/assets/js/components/addons-section.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import List from '@elementor/ui/List';
+import Link from '@elementor/ui/Link';
+import Button from '@elementor/ui/Button';
+import Card from '@elementor/ui/Card';
+import CardActions from '@elementor/ui/CardActions';
+import CardContent from '@elementor/ui/CardContent';
+import CardMedia from '@elementor/ui/CardMedia';
+
+const Addons = ( { ...props } ) => {
+	const addonsArray = props.addonsData.repeater.filter( ( item ) => false === item.is_installed );
```
Comment (hein-obox) [inline modules/home/assets/js/components/addons-section.js:11]:
```suggestion
	const activeAddons = props.addonsData.repeater.filter( ( item ) => false === item.is_installed );
```

File: modules/home/assets/js/components/addons-section.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	const cardsPerRow = 3 === addonsArray.length ? 3 : 2;
```
[the remark this answers is not in the document]
Comment (elchugreeva) [reply modules/home/assets/js/components/addons-section.js:12]: but that is the case only for 3, if it is more, and we start adding rows, we want 2 items in a row, if theres 5 add ons it is 2, 2 and 1

File: modules/home/assets/js/components/addons-section.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+								<CardContent sx={ { display: 'flex', flexDirection: 'column', justifyContent: 'space-between', gap: 3 } }>
+									<Box>
+										<CardMedia image={ item.image } sx={ { height: '58px', width: '58px', mb: 3 } } />
+										<Box>
+											<Typography variant="subtitle2">{ item.title }</Typography>
+											<Typography variant="body2" color="text.secondary">{ item.description }</Typography>
+										</Box>
+									</Box>
+									<CardActions sx={ { p: 0 } }>
+										<Button variant="outlined" size="small" color="promotion" href={ item.url } target="_blank" sx={ { maxWidth: 'fit-content' } }>{ item.button_label }</Button>
```
Comment (hein-obox) [inline modules/home/assets/js/components/addons-section.js:34]:
```suggestion
										<Button variant="outlined" size="small" color="promotion" href={ item.url } target="_blank">{ item.button_label }</Button>
```

File: modules/home/assets/js/components/addons-section.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+									</CardActions>
+								</CardContent>
+							</Card>
+						);
+					} )
+				}
+			</List>
+			<Link variant="body2" color="info.main" underline="none">{ props.addonsData.footer.label }</Link>
```
Comment (hein-obox) [inline modules/home/assets/js/components/addons-section.js:42]: I think the `a href` element is missing here.

File: modules/home/assets/js/components/addons-section.js (hunk 2 above)
Comment (hein-obox) [inline modules/home/assets/js/components/addons-section.js:11]: Not every item has the `is_installed` property.

```suggestion
	const addonsArray = props.addonsData.repeater.filter( ( item ) => true !== item?.is_installed );
```

File: modules/home/assets/js/components/home-screen.js [hunk 6]
```diff
@@ -17,9 +18,10 @@ const HomeScreen = ( props ) => {
 					createNewPageUrl={ props.homeScreenData.create_new_page_url }
 				/>
 				<Box sx={ { display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, justifyContent: 'space-between', gap: 3 } }>
-					<Box sx={ { flex: 1 } }>
+					<Container maxWidth="md" disableGutters={ true } sx={ { display: 'flex', flexDirection: 'column', gap: 3 } }>
```
Comment (hein-obox) [inline modules/home/assets/js/components/home-screen.js:21]: Why do we have a `<Container>` here?
Can't we add the flex styling inside the `<Box>` component?
Or could we use the `<Stack>` instead?

I don't like seeing containers with a maxWidth if we don't really need that.

**Review by ronenboxer — APPROVED** [review]

**Review by hein-obox — APPROVED** [review]

**Pull request merged**